pkg/api/security/security.go

```diff
@@ -102,38 +105,67 @@ func GenerateRootCert(hosts []string, bits int) (
 	return
 }
 
-// FetchAuthToken gets the authentication token from the auth token file & creates one if it doesn't exist
+// FetchAuthToken gets the authentication token from:
+// - configuration (yaml or environment variable)
+// - the auth token file & creates one if it doesn't exist
 // Requires that the config has been set up before calling
 func FetchAuthToken() (string, error) {
 	authTokenFile := filepath.Join(filepath.Dir(config.Datadog.ConfigFileUsed()), authTokenName)
+	_, err := os.Stat(authTokenFile)
```

Comment:
Could we use this err ? If the file is in write only it might be interesting to know it ?
The error is a https://golang.org/src/os/error.go?s=653:716#L11

Reply:
If the file is here, stat doesn't return any error. 
But if we want to check the permissions, we could access to `st.Mode().Perm()`.

However, if the file is write only, we cannot read it during the following operation:
```go	
authTokenRaw, err := ioutil.ReadFile(authTokenFile)
if err != nil {
	return "", fmt.Errorf("unable to access authentication token: %s", err)
}
```
IMHO we can let the previous logic handle this case because the error will be understandable without adding more code:
```text
unable to access authentication token: open /tmp/fake-datadog-etc-939332556/auth_token: permission denied
```